MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
@samuel-gallet did you test your change by running MatrixKitSample? On my side the app crashes on mxk_assetsBundle call. `[bundle URLForResource:@"MatrixKit" withExtension:@"bundle"]` returns nil... (FYI i'm using cocoapods 1.2.1, I observed that the retrived bundle is the main one)